internal/view/live_view.go

```diff
@@ -37,7 +37,7 @@ type LiveView struct {
 }
 
 // NewLiveView returns a live viewer.
-func NewLiveView(app *App, title string, m model.ResourceViewer) *LiveView {
+func NewLiveView(app *App, title string, m model.ResourceViewer, autoRefresh bool) *LiveView {
```

Comment:
I don't think we need to change the signature here. Since the app is passed in we can pull the liveview autorefresh config directly in the view ctor.

Reply:
Right, sure!